src/main/java/org/ieeervce/gatekeeper/Entity/FinalStatus.java

```diff
@@ -0,0 +1,8 @@
+package org.ieeervce.gatekeeper.Entity;
+
+public enum FinalStatus {
+    ACCEPTED,
+    REJECTED,
+    PENDING,
+    CANCELLED
```

Comment:
Well, a user might want to cancel a request after raising one because of realising some mistake,so thought of adding option for them to cancel it to prevent further approval requests...